ord_schema/message_helpers.py

```diff
@@ -520,6 +520,88 @@ def set_compound_smiles(compound, value):
                                    value)
 
 
+def is_transition_metal(atom):
+    """Determines if an atom is a transition metal.
+    
+    Args:
+        atom: The atom in question. Should be of type rdkit.Chem.rdchem.Atom
+        
+    Returns:
+        Boolean for whether the atom is a transition metal.
+    """
+    n = atom.GetAtomicNum()
+    return (n>=22 and n<=29) or (n>=40 and n<=47) or (n>=72 and n<=79)
```

Comment:
This can be expressed more succinctly:

`return 22 <= n <= 29 or 40 <= n <= 47 or 72 <= n <= 79`

Reply:
Changed!